host/docker/aic-manager/scripts/image-entry

```diff
@@ -21,23 +23,41 @@ function mount_system_images() {
 
         name=$(basename $img .img)
         dest=$SYSTEM_IMAGES_MOUNT_DIR/$name
+        mkdir -p $dest
         device=$(mount | grep "on $dest" | cut -d' ' -f1)
         if [[ -n $device ]]
         then
             log "$device has already mounted on $dest, skip"
         else
-            # --direct-io is default on, and set it as read only
-            # busybox does not support --show option, so setup first then find it
-            log "setup loop device for $file"
-            if losetup -f -r $file
+            if [ -f "$SYSTEM_VERITY_IMAGE" ]
```

Comment:
I think the dm-verity is only for system.img, so this line should be
`if [[ $file = "system.img" && -f "$SYSTEM_VERITY_IMAGE" ]]`

otherwise it will make a mess if the $file is vendor.img, oem.img, etc.